.cspell/custom_misc.txt

```diff
@@ -2,6 +2,7 @@ ANNchor
 approximator
 approximators
 cand
+color
```

Comment:
If we need to include it, it might be better placed in the `library_terms` dictionary. Or is there some other workaround to keep it out of the dictionary?

Reply:
@jn177246  @tp832944  This is on main inside of library_terms.txt - pre-commit fails without it. I am not sure there is a way around this - I'll move it for now into library_terms.txt to agree with main